Tools/Scripts/libraries/webkitscmpy/webkitscmpy/local/git.py

```diff
@@ -567,7 +567,7 @@ def commit(self, hash=None, revision=None, identifier=None, branch=None, tag=Non
 
         default_branch = self.default_branch
         parsed_branch_point = None
-        log_format = ['-1', '--no-decorate', '--date=unix'] if include_log else ['-1', '--no-decorate', '--date=unix', '--format=short']
+        log_format = ['-1', '--no-decorate', '--date=unix', '--'] if include_log else ['-1', '--no-decorate', '--date=unix', '--format=short', '--']
```

Comment:
Not super excited about this one, as `--` is not part of format. I'd be adding it everywhere instead.